### PR TITLE
update pvplounge to 0.3.0

### PR DESCRIPTION
**Pull request by deadlynjdj**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41976)

### CODE REVIEW

File: manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PvPLounge.PvPLounge
+PackageVersion: 0.3.0
+PackageLocale: en-US
+Publisher: Digital Ingot, Inc.
+PublisherUrl: https://digitalingot.org/
+PublisherSupportUrl: https://pvplounge.com/help
+Author: Digital Ingot, Inc.
+PackageName: PvPLoungg
```
Comment (OfficialEsco) [inline manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.locale.en-US.yaml:11]:
```suggestion
PackageName: PvPLounge Launcher
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PvPLounge.PvPLounge
+PackageVersion: 0.3.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://assets.pvplounge.com/launcher/PvPLounge%20Launcher%20Setup%200.3.0.exe
+  InstallerSha256: df7039a8564a8b72a755b1f22c3e2ed3360363100a38ae241d21b6943c50563f
```
Comment (ItzLevvie) [inline manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml:10]:
```suggestion
  InstallerSha256: df7039a8564a8b72a755b1f22c3e2ed3360363100a38ae241d21b6943c50563f
  Scope: machine
```

Comment (ItzLevvie) [reply manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml:10]: `C:\Program Files\PvPLounge Launcher`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml (hunk 2 above)
Comment (deadlynjdj) [reply manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml:10]: is it not the same?

Comment (ItzLevvie) [reply manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml:10]: GitHub will show you two changes (InstallerSha256 & Scope) ; only one of them (Scope) will be changed.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/41976#pullrequestreview-862814425

**Review by OfficialEsco — APPROVED** [review]
Cba, since there is a waiver you can update it Levvie

**Pull request merged**